### ISSUE TITLE
Will not install NSP files that fail NCA Signature Check

### ISSUE THREAD

**Issue by CaptainCrunch13** [dezem/AtmoXL-Titel-Installer#11]
Hello. I am using v1.5.4 on OFW 13.0.0, Atmosphere 1.1.1, and Hekate 5.6.2 with the most current sigpatches. I have noticed that installation of NSP files that trigger the “Invalid NCA Signature detected!" warning fails when I choose to go ahead with the installation.

I have determined that this includes homebrew NSP files (like TInfoil) and game NSPs that have been converted from cartridge dumps. Offical eShop NSPs which do not fail the signature check all continue to install fine. Here is the exact error message that displays when the installation fails:

"OpenFileSystemWithld:124: Failed to open filesystem. Make sure your signature patches are up to date and set up properly Error Code: 0x00234c02"

Is this a sigpatch issue? An installer issue you will be able to fix? Please let me know.

Thanks for the great installer and all your work!

**Comment by CaptainCrunch13** [comment]
Actually, after a fair amount of troubleshooting, the problem seems to have been on my end. After reinstalling Atmosphere, the problem has cleared up. Sorry to bother you!

**Comment by markmghali** [comment]
What was your fix? Having the same issue. I did update atmosphere sigpatches, hekate to latest versions. on HOS 12.0.0

**Comment by CaptainCrunch13** [comment]
> What was your fix? Having the same issue. I did update atmosphere sigpatches, hekate to latest versions. on HOS 12.0.0

My configuration seems to be different from yours, but here's what worked for me. When I was having problems, I was on HOS 13.0.0, Atmosphere 1.1.1, Hekate 5.6.2, and the most recent sigpatches posted to GBATemp. I did a few things at the same time and am not sure exactly what did it, but here's what I did. In the end, the problem disappeared.

1) Upgraded to Hekate 5.6.3. This did not solve the problem.
2) Deleted and reinstalled the sigpatches. This did not solve the problem.
3) Downloaded Atmosphere 1.1.1 from the GitHub repository (the version I had installed had come from SDSetup) and installed that version
4) Before rebooting my Switch, deleted and reinstalled the sigpatches again for good measure
5) Rebooted the Switch and tried again. This time it worked.

I'm not sure if it was reinstalling Atmosphere from the GitHub repository that did the trick, or if I'd simply installed the sigpatches improperly both times before, but that was my process and everything works now. Hope that helps!
